### PR TITLE
Add "take action" links

### PR DESCRIPTION
**Pull request by arathjz**
**What:**
- Add "take action" links
- Enable to have links opened in a new tab

**Why:**
Relates to: https://app.asana.com/0/1196225495304457/1196225495304485/f

**How:**
- Implement the dc-header and dc-collapse with the "take action" links
- Include `targetBlank` as a new prop for opening a link on a new tab

### Media:
https://www.loom.com/share/b6d275d8a6034599bb1a5bfc11bbdfef

#### Note:
This will be reflected in the landing page once we merge this and update the package dependency in the `home` project

### CODE REVIEW

File: packages/header-component/src/components/dc-header/dc-dropdown.tsx [hunk 1]
```diff
@@ -77,10 +77,9 @@ export class Dropdown {
           ref={(el) => this.dropdownItems = el}
         >
           {
-            this.items.map(item => (
-              <a href={item.href} class="dropdown-item">
+            this?.items?.map(item => (
+              <a href={item.href} class="dropdown-item" target="_blank">
```
Comment (duranmla) [inline packages/header-component/src/components/dc-header/dc-dropdown.tsx:81]: I will rather do what you have done at packages/header-component/src/components/dc-header/dc-header.tsx in fact even better I would choose to spread the attributes of the link so you will have like:

```
[{"href":"http://debtcollective.org/","text":"About us", "target": "_blank"}, {"href":"http://x.org/","text":"About us"}, {"href":"http://google.org/","text":"About us", "data-testid": "hello_world"}]
```

And in that case the extra attribute will be rendered without extra customisation.

**Review by duranmla — CHANGES_REQUESTED** [review]
I will rather go to spread props of each link item instead individually add then. In the future we will be able to add attributes to links on demand for any use such as GTM, CSS styles or extra a11y

File: packages/header-component/src/components/dc-header/dc-dropdown.tsx (hunk 1 above)
Comment (arathjz) [reply packages/header-component/src/components/dc-header/dc-dropdown.tsx]: Nice idea, I've made an update to spread props in the anchors

**Comment by arathjz** [comment]
@duranmla thanks for the feedback! I've addressed your comments. I'll merge this

**Pull request merged**